legacy/aws-amplify-react/README.md

```diff
@@ -3,3 +3,960 @@
+    if (error) return <h3>Error</h3>;
+    if (loading || !listTodos) return <h3>Loading...</h3>;
+    return <ListView todos={listTodos ? listTodos.items : []} />;
+  }}
+</Connect>
+```
+
+## Mutations
+
+For mutations, a `mutation` function needs to be provided with the `Connect` component. A `mutation` returns a promise that resolves with the result of the GraphQL mutation.
+
+```jsx
+import React, { Component } from 'react';
+import * as mutations from './graphql/mutations';
+import * as queries from './graphql/queries';
+import * as subscriptions from './graphql/subscriptions';
+
+class AddTodo extends Component {
+  constructor(props) {
+    super(props);
+    this.submit = this.submit.bind(this);
+    this.state = {
+      name: '',
+      description: '',
+    };
+  }
+
+  handleChange(name, event) {
+    this.setState({ [name]: event.target.value });
+  }
+
+  async submit() {
+    const { onCreate } = this.props;
+    const input = {
+      name: this.state.name,
+      description: this.state.description,
+    };
+    console.log(input);
+
+    try {
+      await onCreate({ input });
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <input
+          name="name"
+          placeholder="name"
+          onChange={(event) => {
+            this.handleChange('name', event);
+          }}
+        />
+        <input
+          name="description"
+          placeholder="description"
+          onChange={(event) => {
+            this.handleChange('description', event);
+          }}
+        />
+        <button onClick={this.submit}>Add</button>
+      </div>
+    );
+  }
+}
+
+class App extends Component {
+  render() {
+    const ListView = ({ todos }) => (
+      <div>
+        <h3>All Todos</h3>
+        <ul>
+          {todos.map((todo) => (
+            <li key={todo.id}>{todo.name}</li>
+          ))}
+        </ul>
+      </div>
+    );
+
+    return (
+      <div className="App">
+        <Connect mutation={graphqlOperation(mutations.createTodo)}>
+          {({ mutation }) => <AddTodo onCreate={mutation} />}
+        </Connect>
+
+        <Connect
+          query={graphqlOperation(queries.listTodos)}
+          subscription={graphqlOperation(subscriptions.onCreateTodo)}
+          onSubscriptionMsg={(prev, { onCreateTodo }) => {
+            console.log('Subscription data:', onCreateTodo);
+            return prev;
+          }}
+        >
+          {({ data: { listTodos }, loading, error }) => {
+            if (error) return <h3>Error</h3>;
+            if (loading || !listTodos) return <h3>Loading...</h3>;
+            return <ListView todos={listTodos.items} />;
+          }}
+        </Connect>
+      </div>
+    );
+  }
+}
+```
+
+---
+
+# Storage
+
+## Photo Picker
+
+`Picker` is used to pick a file from local device storage. `PhotoPicker` and `TextPicker` components are specific to image and text file types .
+
+![Image](/images/photo_picker_and_code.png)
```

Comment:
I think we can find/replace `/images` and instead use:

```suggestion
![Image](photo_picker_and_code.png)
```

And add https://github.com/aws-amplify/docs/blob/main/public/images/photo_picker_and_code.png _beside_ the `README.md`.

Reply:
👍

Reply:
Fixed
![Screen Shot 2021-11-18 at 2 41 58 PM](https://user-images.githubusercontent.com/6165315/142501461-a3167728-6b90-4a85-ac21-f4dc25157dbe.png)